sdjournal/journal.go

```diff
@@ -651,6 +652,43 @@ func (j *Journal) GetDataValue(field string) (string, error) {
 	return strings.SplitN(val, "=", 2)[1], nil
 }
 
+// GetDataBytes gets the data object associated with a specific field from the
+// current journal entry.
+func (j *Journal) GetDataBytes(field string) ([]byte, error) {
+	sd_journal_get_data, err := j.getFunction("sd_journal_get_data")
```

Comment:
Yeah, of course, will do. I was just not sure which way you prefer (with common code extracted or with more linear code).